Package.swift

```diff
@@ -4,8 +4,8 @@ import PackageDescription
 let package = Package(
     name: "web3.swift",
     platforms: [
-        .iOS(SupportedPlatform.IOSVersion.v11),
-        .macOS(SupportedPlatform.MacOSVersion.v10_12)
+        .iOS(SupportedPlatform.IOSVersion.v14),
+        .macOS(SupportedPlatform.MacOSVersion.v11)
```

Comment:
Increased the minimum version to start modernizing APIs; it also allows to use Combine/ and async/await which greatly reduces work for this specific feature with lots of retries etc.

Also preparing min. version to modernize the APIs in the future

Reply:
Combine and async/await are available on iOS v13

Reply:
Had an issue with `flatMap` versions (somehow Xcode assuming I wanted to use the newer one from 14). Will try fix this if possible and move min. version to 13

Reply:
@dnKaratzas managed to bump version to 13

Reply:
Perfect ;)